torch_xla/csrc/xla_data.h

```diff
@@ -0,0 +1,36 @@
+#ifndef XLA_TORCH_XLA_CSRC_XLA_DATA_H_
+#define XLA_TORCH_XLA_CSRC_XLA_DATA_H_
+
+#include <torch/csrc/lazy/backend/backend_data.h>
+#include <torch/csrc/lazy/backend/backend_device.h>
+#include <torch/csrc/lazy/core/shape.h>
+
+#include "third_party/xla_client/computation_client.h"
+
+namespace torch_xla {
+
+// To be noted, the most appropriate way to adopt BackendData
+// should actually be letting ComputationClient::Data inherit it.
+// Since ComputationClient is within TensorFlow, and TF cannot
+// depend on PyTorch. Therefore, we have this intermediate wrapper.
+class XLAData : public torch::lazy::BackendData {
```

Comment:
`Since ComputationClient is within TensorFlow, and TF cannot depend on PyTorch.` isn't true anymore. @JackCaoG can we flatten the dependency chain here?

We can do this in a follow-up PR if so.

Reply:
yea, if computation client can import pytorch file, we should get rid of all of these `Wrap` and `Unwrap` thing.

Reply:
I can take this action item since I'm already untangling the computation_client build target.